homework1/sample_project/calculator/calc.py

```diff
@@ -0,0 +1,4 @@
+def check_power_of_2(a: int) -> bool:
```

Comment:
check_if_power_of_two
or is_power_of_two
And every function must have doctrings.